src/isotools/_transcriptome_io.py

```diff
@@ -171,6 +171,7 @@ def add_sample_from_csv(self: Transcriptome, coverage_csv_file, transcripts_file
     else:
         chromosomes = self.chromosomes
 
+    logger.info('importing transcripts from %s. Please note transcripts with missing annotations will be skipped.', transcripts_file)
```

Comment:
What do you mean with "missing annotations"?

Reply:
For example, in gtf exported from IsoQuant, there are some transcripts without strand annotation.